CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.4.0
```

Comment:
I definitely think we should go to 1.0.0 soon!

I was thinking about doing a 0.4.0, promoting it through Davis, promoting other teams using it on [dev] and if no hangups came pushing for us to release 1.0 (even if it's the same code).

That was my more conservative plan, assuming that I'd have to push for a 1.0. If you're into releasing 1.0, I'm thrilled to do it now.